test/models/shipit/unknown_repository_test.rb

```diff
@@ -0,0 +1,29 @@
+require 'test_helper'
+
+module Shipit
+  class UnknownRepositoryTest < ActiveSupport::TestCase
+    setup do
+      @unknown_repository = UnknownRepository.new
+    end
+
+    test "owner" do
+      assert_equal "unknown-owner", @unknown_repository.owner
+    end
+
+    test "name" do
+      assert_equal "unknown-name", @unknown_repository.name
+    end
+
+    test "stacks" do
+      assert_equal Stack.none, @unknown_repository.stacks
+    end
+
+    test "http_url" do
+      assert_equal "https://github.com/#{@unknown_repository.owner}/#{@unknown_repository.name}", @unknown_repository.http_url
```

Comment:
Isn’t this test a little useless, being identical to the implementation?

Reply:
Removed in https://github.com/Shopify/shipit-engine/commit/fda7be6c60b2439f84df3cdde8b03bc378ee7d50